tests/risk/AnswerKeyAnnotations.ipynb

```diff
@@ -1,87 +0,0 @@
-{
```

Comment:
:+1:

I can not recall if this powers a link on the forums discussing risk metrics.
I think it is ok to remove, since if that link does break, we can change the link to point to a commitish where this file does exist (if it doesn't already.)